api/src/pcapi/core/users/api.py

```diff
@@ -1395,3 +1403,111 @@ def notify_users_before_deletion_of_suspended_account() -> None:
                 "Could not send email before deletion of suspended account",
                 extra={"user": account.id},
             )
+
+
+def anonymize_user(user: users_models.User, *, force: bool = False) -> None:
+    """
+    Anonymise the given User. If force is True, the function will anonymize the user even if they have an address and
```

Comment:
J'ai l'impression que tu as désormais un mélange de `s` et de `z`, peux-tu harmoniser ?
Il y a aussi des `beneficiary_fraud_check.reason = "Anonymised"`, etc.
```suggestion
    Anonymize the given User. If force is True, the function will anonymize the user even if they have an address and
```